### PR TITLE
New schema for the upload step

### PR DESCRIPTION
**Pull request by av3esha**
Added a new step in approval, custom and cd stages to upload files as runtime input

Testing done:
Copied the generated pipeline yaml in local, and saved the pipeline with the new step and executed the pipeline

### CODE REVIEW

**Review by brijeshdhakar — APPROVED** [review]

File: v0/template.json [hunk 1]
```diff
@@ -15223,127 +15223,11 @@
...
-              },
-              "description" : {
-                "desc" : "This is the description for ServerlessAwsLambdaDeployStepInfo"
-              }
-            }
-          },
           "AzureFunctionDeployStepNode_template" : {
             "title" : "AzureFunctionDeployStepNode_template",
             "type" : "object",
-            "required" : [ "identifier", "name", "type" ],
```
Comment (RajBaviskar) [inline v0/template.json:15341]: why "identifier", "name" is dropping out from required

File: v0/template.json [hunk 2]
```diff
@@ -25648,161 +25630,6 @@
               }
             } ]
           },
-          "AzureFunctionDeployStepInfo" : {
```
Comment (RajBaviskar) [inline v0/template.json:25651]: is this getting dropped?

File: v1/template.json [hunk 3]
```diff
@@ -66979,10 +66659,6 @@
                 "$ref" : "#/definitions/pipeline/steps/common/PluginStepNode"
               }, {
                 "$ref" : "#/definitions/pipeline/steps/common/ActionStepNode"
-              }, {
-                "$ref" : "#/definitions/pipeline/steps/cd/AzureFunctionDeployStepNode"
-              }, {
-                "$ref" : "#/definitions/pipeline/steps/cd/AzureFunctionRollbackStepNode"
```
Comment (RajBaviskar) [inline v1/template.json:66985]: are these getting dropped or moved around?

File: v0/template.json (hunk 2 above)
Comment (brijeshdhakar) [reply v0/template.json:25651]: This was added twice in the File. So duplicate is being removed.

**Review by RajBaviskar — APPROVED** [review]

**Comment by bot2-harness** [comment]
This pull request has been automatically closed due to 30 days of inactivity. If you believe this was done in error, please feel free to reopen the PR or reach out to the BT team for further assistance. Thank you for your contributions!

**Pull request closed without merging**